x/ibc/light-clients/solomachine/types/solomachine.go

```diff
@@ -4,11 +4,15 @@ import (
 	"github.com/tendermint/tendermint/crypto"
 
 	codectypes "github.com/cosmos/cosmos-sdk/codec/types"
+	"github.com/cosmos/cosmos-sdk/x/ibc/exported"
 )
 
 // Interface implementation checks.
 var _, _, _, _ codectypes.UnpackInterfacesMessage = &ClientState{}, &ConsensusState{}, &Header{}, &HeaderData{}
 
+// Data is an interface used for all the signature data bytes proto definitions.
+type Data interface{}
```

Comment:
this is added just so I can return the unmarshaled data in `UnmarshalDataByType`